lib/ansible/modules/cloud/azure/azure_rm_virtualnetworkpeering.py

```diff
@@ -266,12 +265,6 @@ def exec_module(self, **kwargs):
                     self.fail("Virtual network {0} in resource group {1} does not exist!".format(
                         self.virtual_network['name'], self.virtual_network['resource_group']))
 
-                # check if remote vnet exists
```

Comment:
so it's not possible to check if it exists if in different sub?

Reply:
it depends on permission of the service principal in another subs. the network api will throw clear message.